feature_engine/encoding/base_encoder.py

```diff
@@ -144,15 +144,24 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
 
         # check if NaN values were introduced by the encoding
         if X[self.encoder_dict_.keys()].isnull().sum().sum() > 0:
+            # obtain the name(s) of the columns have null values
+            nan_columns = X.columns[X.isnull().any()].tolist()
+            if len(nan_columns) > 1:
+                nan_columns_str = ", ".join(nan_columns)
+            else:
+                nan_columns_str = nan_columns[0]
+
             warnings.warn(
-                "NaN values were introduced in the returned dataframe by the encoder."
+                f"NaN values were detected in the feature(s) titled {nan_columns_str}."
```

Comment:
Could we re-phrase the error sentence to this:

f"NaN values were introduced in the feature(s) {nan_columns_str} during the encoding."

Reply:
I revised the warning sentence to what you suggested and fixed the formatting errors. flake8 is great!

Once a PR is in place, do I have to submit new PRs after I push edits to Git? Or, are the pushes automatically sent to the PR?

It seems that pushes are automatically forwarded to the PR.